spec/warren/handler/broadcast_spec.rb

```diff
@@ -34,7 +34,7 @@
 
     before do
       allow(described_class::Channel).to receive(:new)
-        .with(bun_channel, exchange: 'exchange', routing_key_template: 'test.%s')
+        .with(bun_channel, exchange: 'exchange', routing_key_prefix: 'test')
         .and_return(yielded_chanel)
```

Comment:
Very minor but you have used chanel instead of channel. I checked the full file and it is consistent.

Reply:
Ahh, probably worth fixing. Not as important as a spelling error in a public API, but still.